### PR TITLE
make filter functioning again

### PR DESCRIPTION
**Pull request by ucgJhe**
its now working as expected

### CODE REVIEW

**Comment by wtdcode** [comment]
Why not something like `^(open|brk)`?

**Comment by ucgJhe** [comment]
> Why not something like `^(open|brk)`?

just want do demo the comma thing like we did mention in the description , or we just remove it ?

**Comment by wtdcode** [comment]
> > Why not something like `^(open|brk)`?
> 
> just want to demo the comma thing like we did mention in the description , or we should remove that feature ?

I'm not sure why we have to support comma considering the regex has included that functionality. I suggest removing that.

File: tests/test_qltool.py [hunk 1]
```diff
@@ -43,6 +43,17 @@ def test_qltool_json(self):
         except subprocess.CalledProcessError as e:
             raise RuntimeError("command '{}' return with error (code {}): {}".format(e.cmd, e.returncode, e.output))
 
+    def test_qltool_filter(self):
+        create = [sys.executable, '../qltool', 'run', '-f', '../examples/rootfs/arm_linux/bin/arm_hello', '--rootfs', '../examples/rootfs/arm_linux', '-e', '^(open|brk)', '--log-file', 'filtered.qlog']
```
Comment (wtdcode) [inline tests/test_qltool.py:47]: Why not use pipe here?

Comment (ucgJhe) [reply tests/test_qltool.py]: silly me ~

**Comment by wtdcode** [comment]
Merged, thanks.

**Pull request merged**